### ISSUE TITLE
Having web-gui on makes console annoying to type in

### ISSUE THREAD

**Issue by canthisbe** [vincss/mcsleepingserverstarter#162]
Turning on web-gui by setting webPort such as 80 makes typing in the console difficult once the server has started. You must hold another key then press the key you want to press, only once, then let go and repeat...

**Comment by vincss** [comment]
Yes I know it's a limitation when you're running the server on Windows.

**Comment by william64644** [comment]
I'm sorry, what do you mean with "difficult"? what happens?

**Comment by canthisbe** [comment]
Having web-gui on and starting a Paper server (or its forks) on this program causes the issue. It doesn't happen on fabric or vanilla servers. Presumably this is because Paper has highlighting when typing commands into the console while vanilla doesn't.

**Comment by canthisbe** [comment]
I've stumbled upon a workaround that makes the console usable. Adding -Dterminal.jline=false flag to the run command fixes it for Paper servers at the expense of tab completion, basically stripping it to a vanilla server console. Additionally you can add 
-Dterminal.ansi=true to retain colors in the console, as the other flag disables it by default.

**Comment by vincss** [comment]
Hi !
I tried like this : 
minecraftCommand: "java -jar -Dterminal.jline=false -Dterminal.ansi=true paper.jar nogui"

But it doesn't take all my commands into accounts.
How do you start the server ?

**Comment by canthisbe** [comment]
Yeah I've noticed the server only receives every other command, while server starter receives the other half of commands.
